docker-compose.yml

```diff
@@ -39,9 +42,64 @@ services:
         extra_hosts:
```

Comment:
please explain this bit

Reply:
The _webserver_ container (harvester + Airflow) needs to connect with the _ckan_ container (http://ckan:5000) to read harvest sources and write new datasets. 
This line add the IP to the host file of the _webserver_ container and allow this conection.

Reply:
containers can connect between themselves without having any special setting, docker-compose takes care of that.
See the configuration in ckan-cloud-docker for inspiration.

Reply:
Removed all references to HOST_IP. I was wrong.